src/tsolvers/lasolver/Tableau.cc

```diff
@@ -78,9 +78,7 @@ std::vector<LVRef> Tableau::getNonBasicVars() const {
     std::vector<LVRef> res;
     res.resize(varTypes.size());
     for (unsigned i = 0; i < varTypes.size(); ++i) {
-        if (varTypes[i] == VarType::NONBASIC) {
-            res.push_back(LVRef{i});
-        }
+        if (varTypes[i] == VarType::NONBASIC) { res.push_back(LVRef{i}); }
```

Comment:
Hmm, that's a recommendation from linter?

Reply:
That's the effect of the rule
`AllowShortIfStatementsOnASingleLine: WithoutElse`

I am undecided what is the best setting here.
Should we disallow even these short statements on single line?

Reply:
Yeah, tbh I'm not sure, I suppose both is fine, if we already have this rule then let it be, it saves some space